the-collab-lab/tcl-18-smart-shopping-list
# Ma ac firebase

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

 As a component, I want to be able to read from and write to the Firestore database so that users can persist information.



## Related Issue



#1 

## Acceptance Criteria




## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|   ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before



![image](https://user-images.githubusercontent.com/8689897/103584013-32037080-4eb7-11eb-84d3-239afc81d49b.png)


### After



![image](https://user-images.githubusercontent.com/8689897/103584246-58c1a700-4eb7-11eb-9d0f-4467255d512e.png)


## Testing Steps / QA Criteria



1. Then `npm `to `install `the newly added dependencies locally and `npm start` to launch the app.
2.  Press the button "+" multiple times then sales value increases one by one (For example: 0, 1, 2, 3, ... n). Check the value in Firebase.
3. Press the button "reset" then the sales value resets to zero. Check the value in Firebase.
